src/main.rs

```diff
@@ -54,7 +54,7 @@ fn run() -> Result<()> {
                     env_logger::DEFAULT_FILTER_ENV
                 ))
                 .takes_value(true)
-                .possible_values(&["off", "error", "warn", "info", "debug", "trace"])
+                .possible_values(&["off", "error", "warn", "info", "debug", "trace"]),
```

Comment:
What's this comma doing?

Reply:
Must me coming from `rustfmt`; I thought I stripped all of the formatting changes out but I must have missed that. 🤷🏻‍♂️